src/BudgetManager/BudgetManager.Model/CurrencyRates.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using BudgetManager.Shared.Models.MongoDB.Models.Interfaces;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDbGenericRepository.Attributes;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace BudgetManager.Model
+{
+    [CollectionName("CurrencyRates")]
+    public class CurrencyRates : IModelBase
+    {
+        [JsonIgnore]
+        [BsonId]
+        public Guid Id { get; set; } = new Guid("66c05d55-dc7a-4d91-a965-a037db973b06");
```

Comment:
Can you change storing Id to not hardcode?
```suggestion
        public Guid Id { get; set; } = Guid.NewGuid;
```

Reply:
Nope, The Id should always remain the same as there will only ever be one entity and having one Id that doesn't change makes upserting possible.